gp-includes/things/translation.php

```diff
@@ -252,6 +252,17 @@ public function prepare_fields_for_save( $args ) {
 		if ( is_array( gp_array_get( $args, 'warnings' ) ) ) {
 			$args['warnings'] = serialize( $args['warnings'] );
 		}
+
+		/**
+		 * Fires after a translation is preparing for save.
```

Comment:
`Fires after a` is only used for actions, not filters. For filters we use `Filters the…`